sonar-vbnet-plugin/src/main/java/org/sonar/plugins/vbnet/VbNetModuleConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SonarVB
+ * Copyright (C) 2012-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.vbnet;
+
+import org.sonar.api.config.Configuration;
+import org.sonarsource.dotnet.shared.plugins.AbstractModuleConfiguration;
+
+@Deprecated
+public class VbNetModuleConfiguration extends AbstractModuleConfiguration {
+  public VbNetModuleConfiguration(Configuration configuration) {
```

Comment:
coverage is failing for this type of classes - you could add a test to check which type of instantiation it is using?

hm it's a bit annoying because this is boiler plate code

Reply:
actually you should test that it passes the correct `LANGUAGE_KEY`